session/tidb.go

```diff
@@ -214,7 +214,7 @@ func runStmt(ctx context.Context, sctx sessionctx.Context, s sqlexec.Statement)
 		// If it is not a select statement, we record its slow log here,
 		// then it could include the transaction commit time.
 		if rs == nil {
-			s.(*executor.ExecStmt).LogSlowQuery(se.GetSessionVars().TxnCtx.StartTS, err != nil)
+			s.(*executor.ExecStmt).LogSlowQuery(se.GetSessionVars().TxnCtx.StartTS, err == nil)
```

Comment:
please add a UT.

Reply:
It's hard to add UT to test this..., any suggestion？

Reply:
how about duplicated primary key error?

Reply:
? To test this problem, we need the `slow log` output and  get the SQL corresponding slow log, then check the `succ` field. 

The above method is hard to implement in UT.